lesson08/assignment/vue-project/src/components/Accordion.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <!-- Container  -->
+  <b-container class="accordion">
+    <h2 class="pt-5 display-4 text-muted">{{ header }}</h2>
+    <p class="lead mb-4 text-muted">{{ subtitle }}</p>
+
+    <!-- Loop thru an array of items and dynamically generate accordion/cards with content and images  -->
+    <b-card no-body class="mb-1" v-for="item in items" v-bind:key="item.id">
+      <b-card-header header-tag="header" class="p-1" role="tab">
+        <!-- concat item ID to generate unique ID for each accordion/tabs to be toggled separately-->
+        <b-button block v-b-toggle="'item-' + item.id" variant="info">{{ item.title}}</b-button>
+      </b-card-header>
+      <b-collapse :id="'item-' + item.id" accordion="my-accordion" role="tabpanel">
+        <b-card-body>
+          <!-- using dot notation to access data in array of items -->
+          <img
+            :src="item.imgUrl"
+            :alt="item.imgAlt"
+            class="img-fluid float-left"
+            style="max-width: 22rem;"
```

Comment:
Inline styles, even in components like this, are not best practice. I'd recommend adding a class of something like "med-img", and creating a max-width rule for it in CSS.